batchupload/make_info.py

```diff
@@ -245,27 +247,36 @@ def handle_args(args):
                 options['in_file'] = common.convert_from_commandline(value)
             elif option == '-base_name':
                 options['base_name'] = common.convert_from_commandline(value)
+            elif option == '-update_mappings':
+                if value.lower() in ('f', 'false', 'no', 'n'):
```

Comment:
I've added a helper method instead which will also protest if it gets something unexpected.